debian/control

```diff
@@ -9,7 +9,7 @@ Homepage: http://www.cloudstack.org/
 
 Package: cloudstack-common
 Architecture: all
-Depends: ${misc:Depends}, ${python3:Depends}, genisoimage, nfs-common, python3-pip, python3-distutils, python3-netaddr
+Depends: ${misc:Depends}, ${python3:Depends}, genisoimage, nfs-common, python3-pip, python3-distutils-extra, python3-netaddr
```

Comment:
```suggestion
Depends: ${misc:Depends}, ${python3:Depends}, genisoimage, nfs-common, python3-pip, python3-distutils | python3-distutils-extra, python3-netaddr
```
@davidjumani will this affect other Ubuntu releases? Saw this other PR, not sure if adding pipe will work here

Reply:
@shwstppr Thanks. I can test with these changes. Am currently running tests on Ubuntu 18 to check if everything works fine!